### DISCUSSION TITLE
Any possibility to use gademe without internet?

### DISCUSSION THREAD

**Discussion by GuillemFo** [JCluzet/42_EXAM#102] [Ideas]
Is there any command to relaunch grademe so u don't need to run the: bash -c "$(curl https://grademe.fr)" line?
Im thinking of a way to use the already downloaded repo to run the exam, like a command or something to launch it again. Be able to clear all done tests and the possibility to create new folders to start new tests??.
The idea is to use the repo with no internet.

Just an idea ;)

**Comment by JCluzet** [comment]
Hi, you can use 42_EXAM simply by downloading the repo, then doing make inside. You don't necessarily need the Grademe command :)

**Reply by GuillemFo** [reply]
thanks, I did not read the makefile at all  😅👍
